packages/amplify-category-function/src/provider-utils/awscloudformation/utils/getDependentFunction.ts

```diff
@@ -1,9 +1,12 @@
 import { $TSContext, $TSObject } from 'amplify-cli-core';
 import * as path from 'path';
-import { loadFunctionParameters } from './loadFunctionParameters';
+import { loadFunctionParameters, loadFunctionStackAsJSON } from './loadFunctionParameters';
 import { ServiceName } from './constants';
 import { categoryName } from '../../../constants';
 
+/**
+ * @deprecated in favor of lambdasWithMissingApiDependency, using a list of existing models rather than deleted ones
+ */
 export async function lambdasWithApiDependency(
```

Comment:
Can this be removed now?

Reply:
Yes, deleting this just forgot to delete it after removing references